editor/editor_vcs_interface.cpp

```diff
@@ -152,6 +255,191 @@ String EditorVCSInterface::get_vcs_name() {
+	ClassDB::bind_method(D_METHOD("_stage_file", "file_path"), &EditorVCSInterface::_stage_file);
+	ClassDB::bind_method(D_METHOD("_unstage_file", "file_path"), &EditorVCSInterface::_unstage_file);
+	ClassDB::bind_method(D_METHOD("_discard_file", "file_path"), &EditorVCSInterface::_discard_file);
+	ClassDB::bind_method(D_METHOD("_get_previous_commits"), &EditorVCSInterface::_get_previous_commits);
+	ClassDB::bind_method(D_METHOD("_get_branch_list"), &EditorVCSInterface::_get_branch_list);
+	ClassDB::bind_method(D_METHOD("_checkout_branch", "branch"), &EditorVCSInterface::_checkout_branch);
+	ClassDB::bind_method(D_METHOD("_push"), &EditorVCSInterface::_push);
+	ClassDB::bind_method(D_METHOD("_pull"), &EditorVCSInterface::_pull);
+	ClassDB::bind_method(D_METHOD("_fetch"), &EditorVCSInterface::_fetch);
+	ClassDB::bind_method(D_METHOD("_set_up_credentials"), &EditorVCSInterface::_set_up_credentials);
+	// ClassDB::bind_method(D_METHOD("_get_line_diff"), &EditorVCSInterface::_get_line_diff);
+
+	ClassDB::bind_method(D_METHOD("is_plugin_ready"), &EditorVCSInterface::is_plugin_ready);
+
+	// API methods that redirect calls to the proxy end points
+	ClassDB::bind_method(D_METHOD("initialize", "project_root_path"), &EditorVCSInterface::initialize);
+	ClassDB::bind_method(D_METHOD("is_vcs_initialized"), &EditorVCSInterface::is_vcs_initialized);
+	// ClassDB::bind_method(D_METHOD("get_modified_files_data"), &EditorVCSInterface::get_modified_files_data);
+	ClassDB::bind_method(D_METHOD("stage_file", "file_path"), &EditorVCSInterface::stage_file);
+	ClassDB::bind_method(D_METHOD("unstage_file", "file_path"), &EditorVCSInterface::unstage_file);
+	ClassDB::bind_method(D_METHOD("discard_file", "file_path"), &EditorVCSInterface::discard_file);
+	ClassDB::bind_method(D_METHOD("commit", "msg"), &EditorVCSInterface::commit);
+	// ClassDB::bind_method(D_METHOD("get_file_diff", "file_path", "area"), &EditorVCSInterface::get_file_diff);
+	ClassDB::bind_method(D_METHOD("shut_down"), &EditorVCSInterface::shut_down);
+	ClassDB::bind_method(D_METHOD("get_project_name"), &EditorVCSInterface::get_project_name);
+	ClassDB::bind_method(D_METHOD("get_vcs_name"), &EditorVCSInterface::get_vcs_name);
+	// ClassDB::bind_method(D_METHOD("get_previous_commits"), &EditorVCSInterface::get_previous_commits);
+	// ClassDB::bind_method(D_METHOD("get_branch_list"), &EditorVCSInterface::get_branch_list);
+	ClassDB::bind_method(D_METHOD("checkout_branch", "branch"), &EditorVCSInterface::checkout_branch);
+	ClassDB::bind_method(D_METHOD("push"), &EditorVCSInterface::push);
+	ClassDB::bind_method(D_METHOD("pull"), &EditorVCSInterface::pull);
+	ClassDB::bind_method(D_METHOD("fetch"), &EditorVCSInterface::fetch);
+	ClassDB::bind_method(D_METHOD("set_up_credentials"), &EditorVCSInterface::set_up_credentials);
+	// ClassDB::bind_method(D_METHOD("get_line_diff"), &EditorVCSInterface::get_line_diff);
```

Comment:
Are the commented bindings needed?

Reply:
Thanks for point it out. Those are need and used. I might have messed while rebasing.